app/src/main/java/fr/free/nrw/commons/leaderboard/LeaderboardPagerAdapter.java

```diff
@@ -0,0 +1,52 @@
+package fr.free.nrw.commons.leaderboard;
```

Comment:
@silkypriya it would be great if you can reuse  https://github.com/commons-app/apps-android-commons/blob/master/app/src/main/java/fr/free/nrw/commons/explore/ViewPagerAdapter.java
instead of this file. Most of the code seems to be common in both of these files.

Reply:
Hello @ujjwalagrawal17, I've made the required changes. Kindly review. Thank you!